src/servers/src/http/influxdb.rs

```diff
@@ -40,7 +39,6 @@ pub async fn influxdb_health() -> Result<impl IntoResponse> {
 }
 
 #[axum_macros::debug_handler]
-#[tracing::instrument(skip_all, fields(protocol = "influxdb", request_type = "write_v1"))]
```

Comment:
BTW, why do you remove these instrument attrs?

Reply:
hmm... must be something accidental, I'll restore that back